docs/pipelineruns.md

```diff
@@ -185,6 +185,10 @@ to all `persistentVolumeClaims` generated internally.
 You can specify `Parameters` that you want to pass to the `Pipeline` during execution,
 including different values of the same parameter for different `Tasks` in the `Pipeline`.
 
+You must specify all the `Parameters` that the `Pipeline` expects. 
+
+You can pass in extra `Parameters` if needed depending on you use case. 
```

Comment:
typo: your use case

(you might want to elaborate on "your use case" here, e.g. give an example where a CI system sets up some infrastructure and wants to pass all the configuration values to all pipelineruns, or use #2513 for inspiration)